workflows/dragen-transcriptome-pipeline/3.9.3/dragen-transcriptome-pipeline__3.9.3.cwl

```diff
@@ -321,7 +323,13 @@ steps:
       dummy_file:
         source: create_dummy_file_step/dummy_file_output
       cl_config:
-        source: cl_config
+        source:
+          - cl_config
+          - output_file_prefix
+        valueFrom: |
+          ${
+            return add_sample_to_sample_name_replace_in_multiqc_cl_config(self[0], self[1] + "_qualimap", self[1]);
+          }
```

Comment:
was this tested on ICA?

Reply:
Yep, see this comment  - https://github.com/umccr/cwl-ica/pull/281#issuecomment-1433932756